src/commands/appSettings/AppSettingsClient.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IAppSettingsClient } from 'vscode-azureappservice';
+import { ISubscriptionContext } from 'vscode-azureextensionui';
+import { EnvironmentTreeItem } from '../../tree/EnvironmentTreeItem';
+import { requestUtils } from '../../utils/requestUtils';
+
+export class AppSettingsClient implements IAppSettingsClient {
+
+    public fullName: string;
+    public isLinux: boolean;
```

Comment:
I don't see you setting `isLinux` in the constructor, but the type here makes it seem required